ta/arch/arm/link.mk

```diff
@@ -31,6 +31,9 @@ link-ldflags += -T $(link-script-pp$(sm))
 link-ldflags += -Map=$(link-out-dir$(sm))/$(user-ta-uuid).map
 link-ldflags += --sort-section=alignment
 link-ldflags += -z max-page-size=4096 # OP-TEE always uses 4K alignment
+ifeq ($(sm)-$(CFG_TA_BTI),ta_arm64-y)
+#link-ldflags += $(call ld-option,-z force-bti) --fatal-warnings
```

Comment:
Is this intentional?

Reply:
I missed this one. @jforissier, I don't see any issues if I uncomment it. Any reason for commenting it in this patch ?

Reply:
No, sorry about that. It is a leftover from my testing (I did not want `tee.elf` to fail linking in order to be able to test the TA part). You can remove the comment.